src/routes/safe/components/Transactions/TxList/TxDataRow.tsx

```diff
@@ -21,15 +22,28 @@ type TxDataRowType = {
   hasExplorer?: boolean
   title: string
   value?: string
+  isArray?: boolean
+  method?: string
+  paramType?: string
 }
 
-export const TxDataRow = ({ children, inlineType, hasExplorer = true, title, value }: TxDataRowType): ReactElement => (
+export const TxDataRow = ({
+  children,
+  inlineType,
+  hasExplorer = true,
+  title,
+  value,
+  isArray,
+  method,
+  paramType,
+}: TxDataRowType): ReactElement => (
   <StyledGridRow>
-    <Text size="xl" as="span">
+    <Text size="xl" as="span" color="placeHolder">
       {title}
     </Text>
+    {isArray && value && method && paramType && <Value method={method} type={paramType} value={value} />}
     {value && inlineType === 'hash' && (
-      <InlineEthHashInfo
+      <InlinePrefixedEthHashInfo
```

Comment:
You sure it should be prefixed?

Reply:
Only address type params should be prefixed. Thanks for catching that